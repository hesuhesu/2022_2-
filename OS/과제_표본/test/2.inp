50
33777 Melaine Pinckney
14266 Bobbi Doyle
25920 Ezra Dibiase
22293 Lynwood Liston
35505 Corinna Izzi
18972 Vikki Blackmon
17148 Sudie Lamothe
00704 Todd Camarillo
08035 Rodger Gunderson
40876 Genna Bussell
31259 Jermaine Graig
09293 Charlesetta Bramblett
06069 Xiomara Zielinski
48637 Elois Printup
27131 Kirstie Allen
21499 Coy Costello
25284 Denae Corbeil
03794 Cristina Gibney
19973 Nada Ruge
16322 Sherry Rambin
11660 Benny Dimattia
12548 Cherie Larsen
28944 Marvis Mancia
41747 Shena Livengood
07818 Julene Valcourt
26998 Ali Parillo
33034 Thersa Cocke
11238 Everett Scroggins
25872 Jolanda Plain
37465 Geneva Wilkinson
16787 Mindy Aycock
24064 Jesenia Starner
12625 Victor Blade
13411 Marlo Raney
48726 Tonja Trepanier
05083 Faustino Haliburton
30607 Mariana Gartman
38830 Kristal Illingworth
15309 Sonia Mctaggart
13475 Amalia Tillery
18272 Troy Reichert
42412 Latasha Pallas
46754 Harmony Brookins
01907 Palma Lehto
39768 Larhonda Lovering
23748 Marcie Hommel
45182 Keira Walburn
43522 Sadie Julien
15635 Sherika Towers
05795 Ashley Briscoe
