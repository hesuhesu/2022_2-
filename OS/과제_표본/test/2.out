00704 Todd        Camarillo
01907 Palma       Lehto
03794 Cristina    Gibney
05083 Faustino    Haliburton
05795 Ashley      Briscoe
06069 Xiomara     Zielinski
07818 Julene      Valcourt
08035 Rodger      Gunderson
09293 Charlesetta Bramblett
11238 Everett     Scroggins
11660 Benny       Dimattia
12548 Cherie      Larsen
12625 Victor      Blade
13411 Marlo       Raney
13475 Amalia      Tillery
14266 Bobbi       Doyle
15309 Sonia       Mctaggart
15635 Sherika     Towers
16322 Sherry      Rambin
16787 Mindy       Aycock
17148 Sudie       Lamothe
18272 Troy        Reichert
18972 Vikki       Blackmon
19973 Nada        Ruge
21499 Coy         Costello
22293 Lynwood     Liston
23748 Marcie      Hommel
24064 Jesenia     Starner
25284 Denae       Corbeil
25872 Jolanda     Plain
25920 Ezra        Dibiase
26998 Ali         Parillo
27131 Kirstie     Allen
28944 Marvis      Mancia
30607 Mariana     Gartman
31259 Jermaine    Graig
33034 Thersa      Cocke
33777 Melaine     Pinckney
35505 Corinna     Izzi
37465 Geneva      Wilkinson
38830 Kristal     Illingworth
39768 Larhonda    Lovering
40876 Genna       Bussell
41747 Shena       Livengood
42412 Latasha     Pallas
43522 Sadie       Julien
45182 Keira       Walburn
46754 Harmony     Brookins
48637 Elois       Printup
48726 Tonja       Trepanier
